extensions/sql-database-projects/src/controllers/mainController.ts

```diff
@@ -102,7 +103,7 @@ export default class MainController implements vscode.Disposable {
 					isNewTable: false,
 					tableScriptPath: filePath,
 					projectFilePath: projectPath,
-					allScripts: projectNode.project.files.map(entry => entry.fsUri.fsPath),
+					allScripts: projectNode.project.files.filter(entry => entry.type === EntryType.File).map(entry => entry.fsUri.fsPath),
```

Comment:
The project files are all the files either included by the default *.sql glob for SDK-style projects or files included with a `<Build Include="file.sql">`. People could include other file types if they like, but then the build would fail. It wouldn't hurt to add filtering for only .sql files though, so at least they could still open the table designer even if build would fail because there's a Build Include with a .txt file on accident.

Reply:
that's exactly what I'm thinking, filtering on `*.sql` files makes table designer more robust.